pkg/virt-handler/vm.go

```diff
@@ -1938,7 +1938,7 @@ func (d *VirtualMachineController) checkVolumesForMigration(vmi *v1.VirtualMachi
 				return blockMigrate, err
 			}
 			if !shared {
-				return true, fmt.Errorf("cannot migrate VMI with non-shared PVCs")
+				return true, fmt.Errorf("cannot migrate VMI: PVC %v is not shared, live migration requires that all PVCs must be shared (using RWX access mode)", volName)
```

Comment:
Can we use `ReadWriteMany` instead of `RWX`? I know that this is the kubectl shortcut, but the API or yamls don't have this.

Reply:
done